setup/docker/README.txt

```diff
@@ -0,0 +1,49 @@
+
```

Comment:
Is this file required? I would prefer to put information on the official website.